idunn/utils/default_settings.yaml

```diff
@@ -1,3 +1,6 @@
 
 ES_URL: http://localhost:9200/
 
+WIKI_USER_AGENT: 'Idunn' # Used in requests to external wiki* APIs
+
+DEFAULT_LANGUAGE: 'en' # Fallback when no 'lang' in request
```

Comment:
So are we sure we don't want to force to get the lang ?

Reply:
I am not convinced myself. This defines a default value upstream in the handler function, so that blocks implementations no longer need to handle None in `lang`.  
But we might need other language fallbacks in the future, when we'll derive the language from HTTP headers (that may contain multiple languages)